### DISCUSSION TITLE
History Graph: Sonstige InfluxDB Graphen

### DISCUSSION THREAD

**Discussion by Sineos** [Zefau/ioBroker.jarvis#2045] [Q&A] [answered]
Gibt es die Möglichkeit auch weitere InfluxDB Abfragen als Datengrundlage zu nutzen? Sprich Abfragen, die keinen History-DP in ioBroker haben, aber in Influx vorhanden sind.

**Comment by Zefau** [comment, marked as answer]
Nein. Jarvis ist eine Visualisierung und eigl keine Datenverarbeitung. Du müsstest es in ioBroker abfragen, verarbeiten und in einen State schreiben
